Util/RequestUtil.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Util;
+
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * Utility class for request-related functions.
+ *
+ * @internal Should not be called by user code
+ *
+ * @author Florian Voutzinos <florian@voutzinos.com>
+ */
+final class RequestUtil
+{
+    /**
+     * Checks if the request is a form post.
+     *
+     * @param Request $request
+     *
+     * @return boolean
+     */
+    public static function isFormPost(Request $request)
+    {
+        if (!$request->isMethod('POST')) {
```

Comment:
Indeed these cases are missing. I will move this code to a private listener method as @lsmith77 suggested and add extra tests for this